scripts/run_docker_build.sh

```diff
@@ -57,5 +57,5 @@ find conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt"
     | xargs -n1 cat | grep -v -e "^#" -e "^$" | \
     xargs -r yum install -y
 
-conda-build-all /conda-recipes --matrix-conditions "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all /conda-recipes --matrix-conditions "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
I don't think we should modify the scripts dir in a recipe PR. @conda-forge/core may know more.